lib/auth/grpcserver.go

```diff
@@ -2508,6 +2508,14 @@ func validateUserSingleUseCertRequest(ctx context.Context, actx *grpcContext, re
 		return trace.BadParameter("unknown certificate Usage %q", req.Usage)
 	}
 
+	if req.RequesterName == proto.UserCertsRequest_TSH_DB_LOCAL_PROXY_TUNNEL &&
+		req.Usage == proto.UserCertsRequest_Database {
```

Comment:
Small nit but I think it's easier to read when we first check `req.Usage` and only then `req.RequesterName`. `UserCertsRequest_TSH_DB_LOCAL_PROXY_TUNNEL` is not relevant for any other usage than the db one.

```suggestion
	if req.Usage == proto.UserCertsRequest_Database &&
		req.RequesterName == proto.UserCertsRequest_TSH_DB_LOCAL_PROXY_TUNNEL {
```